ex_2.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Jun 10 08:52:42 2018
+
+@author: amyhaddad
+"""
+
+print("What is your favorite quote?", end= ' ')
```

Comment:
I learned something new that that makes the default ending of a `print` statement  a space and not a line break. :)